inz-demo/id-card/Issues/id-card/description.xml

```diff
@@ -0,0 +1,128 @@
+<IssueSpecification version="4">
+    <Name>
+        <en>Demo ID Card</en>
+        <nl>Demo ID Card</nl>
+    </Name>
+    <ShortName>
+        <en>ID Card</en>
+        <nl>ID Card</nl>
+    </ShortName>
+    <SchemeManager>inz-demo</SchemeManager>
+    <IssuerID>id-card</IssuerID>
+    <CredentialID>ID Card</CredentialID>
+    <Description>
+        <en>ID Card</en>
+        <nl>ID Card</nl>
+    </Description>
+    <ShouldBeSingleton>true</ShouldBeSingleton>
+    <IssueURL>
+        <en>https://demo.irma.nz/id-card/</en>
+        <nl>https://demo.irma.nz/id-card/</nl>
+    </IssueURL>
+    <Category>
+        <en>Personal</en>
+        <nl>Persoonlijk</nl>
+    </Category>
+    <Attributes>
+      <Attribute id="id">
+          <Name>
+              <en>Card ID Number</en>
+              <nl>Card ID Number</nl>
+          </Name>
+          <Description>
+              <en>Your card number</en>
+              <nl>Your card number</nl>
+          </Description>
+      </Attribute>
+      <Attribute id="issueDate">
+          <Name>
+              <en>Card Issue Date</en>
+              <nl>Card Issue Date</nl>
+          </Name>
+          <Description>
+              <en>The issue date of the ID Card</en>
+              <nl>The issue date of the ID Card</nl>
+          </Description>
+      </Attribute>
+      <Attribute id="expiryDate">
+          <Name>
+              <en>Card Expiry Date</en>
+              <nl>Card Expiry Date</nl>
+          </Name>
+          <Description>
+              <en>The expiry date of the Id Card</en>
+              <nl>The expiry date of the Id Card</nl>
+          </Description>
+      </Attribute>
+        <Attribute id="over18">
+            <Name>
+                <en>Over 18</en>
+                <nl>Over 18</nl>
+            </Name>
+            <Description>
+                <en>Over 18</en>
+                <nl>Over 18</nl>
+            </Description>
+        </Attribute>
+        <Attribute id="over20">
+            <Name>
+                <en>Over 20</en>
+                <nl>Over 20</nl>
+            </Name>
+            <Description>
+                <en>Over 20</en>
+                <nl>Over 20</nl>
+            </Description>
+        </Attribute>
+        <Attribute id="over65">
+            <Name>
+                <en>Over 65</en>
+                <nl>Over 65</nl>
+            </Name>
+            <Description>
+                <en>Over 65</en>
+                <nl>Over 65</nl>
+            </Description>
+        </Attribute>
```

Comment:
Let's move over fields to the end, as they are derived values. I propose this ordering:
- First Name
- Last Name
- Full Name
- Birthdate
- Over 18
- Over 65
- Expiry Date
- Issue Date
- Number